### PR TITLE
V2 implementation

### PR DESCRIPTION
**Pull request by b-siva**

### CODE REVIEW

File: lib/stubhub/client.rb [hunk 1]
```diff
@@ -53,65 +53,71 @@ def login(opts = {})
...
+          externalListingId: opts[:external_id]
           # status: 'INACTIVE' # Disable for production
       }
 
       opts[:traits].each do |trait|
-        listing_params[:ticketTraits].push({id: trait.to_s})
+        listing[:ticketTraits].push({id: trait.to_s,operation: "ADD"})
       end
-
+      puts listing
```
Comment (b-siva) [inline lib/stubhub/client.rb]: remove this puts.

File: lib/stubhub/client.rb [hunk 2]
```diff
@@ -519,6 +537,8 @@ def delete(path)
       headers['Authorization'] = "Bearer #{self.access_token}"
 
       response = self.class.delete(path, options)
+      
+      puts "Stubhub DELETE response #{response.body}"
```
Comment (b-siva) [inline lib/stubhub/client.rb:549]: remove this puts

File: lib/stubhub/client.rb [hunk 3]
```diff
@@ -554,7 +574,6 @@ def post(path, type, body)
       end
 
       puts "Stubhub POST #{path} #{options}"
```
Comment (b-siva) [inline lib/stubhub/client.rb:584]: remove this puts

File: lib/stubhub/client.rb [hunk 4]
```diff
@@ -53,65 +53,71 @@ def login(opts = {})
...
+        listing[:splitQuantity] = opts[:split_option]
       end
 
       if opts[:in_hand_date]
-        listing_params[:inhandDate] = opts[:in_hand_date]
-      end
-
-      if opts.include? :tickets
-         url = "/inventory/listings/v1/barcodes"
-         listing_params[:tickets] = opts[:tickets]
```
Comment (b-siva) [inline lib/stubhub/client.rb]: what about this tickets logic which changes the "v1/barcodes". Is it not needed anymore?

Comment (santoshkumar611) [reply lib/stubhub/client.rb]: yes but need to change according to v2

**Pull request merged**